msktutil.cpp

```diff
@@ -164,8 +164,9 @@ void qualify_principal_vec(std::vector<std::string> &principals,
     }
 }
 
-
-int finalize_exec(msktutil_exec *exec, msktutil_flags *flags)
+// Returns an LDAPConnection, unless we are in cleanup mode
+// NULL return are error, unless we are in cleanup mode
+LDAPConnection *finalize_exec(msktutil_exec *exec, msktutil_flags *flags)
```

Comment:
This one confuses me. The method name does not return match to the return value.